src/components/flat-table/flat-table.stories.mdx

```diff
@@ -2094,7 +2094,7 @@ of the button control.
 <Canvas>
```

Comment:
Yet again, not to sort in this PR (unless you really, really, REALLY want to 🤷🏻 ) but I've just noticed a bug with the cursor in this story that also exists in `master`. Let me know and I'll create a new ticket for this issue too 👍🏻 

![flat-table-draggable-bug](https://user-images.githubusercontent.com/56251247/150522575-e898205d-a618-4073-82ed-2a48f2c41a46.gif)